src/util/symbols.js

```diff
@@ -1,9 +1,13 @@
-export const connected = '____skate_connected';
-export const created = '____skate_created';
-export const name = '____skate_name';
-export const props = '____skate_props';
-export const ref = '____skate_ref';
-export const renderer = '____skate_renderer';
-export const rendering = '____skate_rendering';
-export const rendererDebounced = '____skate_rendererDebounced';
-export const shadowRoot = '____skate_shadowRoot';
+function createSymbol(name) {
```

Comment:
@treshugart Actually, do we need to add a description to a `Symbol`?  Is it mostly useful for debugging, but if we create a random string, and you are in a context where Symbol is not supported, it will be difficult to know. Maybe we can keep those since they are more descriptive ?